src/orquestra/sdk/_base/_driver/_client.py

```diff
@@ -657,23 +658,19 @@ def get_workflow_run_logs(self, wf_run_id: _models.WorkflowRunID) -> List[str]:
         decoded = untarred.read().decode()
 
         # Parse the decoded data as logs
-        # TODO: index by taskinvocationID rather than workflowrunID [ORQSDK-777]
-        logs = []
-        for section in decoded.split("\n"):
-            if len(section) < 1:
+        # TODO: index by taskinvocationID rather than workflowrunID [ORQSDK-840]
+        # logs = []
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏commented out code